lnbits/wallets/lndrest.py

```diff
@@ -195,6 +195,8 @@ async def get_payment_status(self, checking_id: str) -> PaymentStatus:
                             if "message" in line["error"]
                             else line["error"]
                         )
+                        if line["error"]["code"] == 5 and line["error"]["message"] == "payment isn't initiated":
```

Comment:
I think the following is more correct:

```suggestion
                        if line["error"].get("code") == 5 and line["error"].get("message") == "payment isn't initiated":
```